app/src/main/java/com/nextcloud/talk/activities/CallActivity.java

```diff
@@ -2591,12 +2593,16 @@ private void playCallingSound() {
 
     private void stopCallingSound() {
         if (mediaPlayer != null) {
-            if (mediaPlayer.isPlaying()) {
-                mediaPlayer.stop();
+            try {
+                if (mediaPlayer.isPlaying()) {
+                    mediaPlayer.stop();
+                }
+            } catch (IllegalStateException e) {
+                Log.e(TAG, "mediaPlayer was not initialized", e);
+            } finally {
+                mediaPlayer.release();
```

Comment:
Should check for `null` here as well, as the field may have become `null` at this point.

Reply:
Redundant nullcheck of com.nextcloud.talk.activities.CallActivity.mediaPlayer, which is known to be non-null in com.nextcloud.talk.activities.CallActivity.stopCallingSound()

https://www.kaminsky.me/nc-dev/talk-android-findbugs/920.html

Reply:
But it's not known? it's a field, which means something else could change it in between... Either way, I don't think it's that important, and will be "fixed" when this is ported to Kotlin anyway :)